### DISCUSSION TITLE
DIA NN for Tims TOF data

### DISCUSSION THREAD

**Discussion by jeewan-rijal** [vdemichev/DiaNN#119] [General]
Hello,
I am trying to use DIA NN (version 1.7.4) for analyzing DIA data from Tims TOF pro. I noticed that .D raw files generated by Tims TOF cant be added.
Is there any other way to load the raw files generated using Tims TOF pro?
Thanks in advance.

**Comment by vdemichev** [comment]
Please use DIA-NN 1.7.16, it fully support timsTOF Pro data.

Vadim

**Reply by jeewan-rijal** [reply]
Oh, Ok thanks for the response, I will try with the version mentioned.
